image_bootstrap/patches/cloud-init-0-7-6-pkgbuild.patch

```diff
@@ -33,6 +33,9 @@
 +    sed -i '1s/python$/&2/' $pkgname-$PKGVER/tools/read*
 +    sed -i 's:/etc/systemd:/usr/lib/systemd:g' $pkgname-$PKGVER/setup.py
 +    sed -i 's,name: ubuntu,name: archlinux,;s,gecos: Ubuntu,gecos: Archlinux,' $srcdir/$pkgname-$PKGVER/config/cloud.cfg
++
++    cd ${srcdir}/${pkgname}-${PKGVER}
++    patch -Np1 -i "${OLDPWD}"/../cloud-init-0-7-6-uid.patch
```

Comment:
You can use `SKIP` as a special hash value to avoid having to include it.

All the functions in a `PKGBUILD` start out in `$srcdir`, so
```
cd $pkgname-$pkgver
patch -Np1 -i ../fix-logging.patch
```
effectively also refers to `$srcdir/fix-logging.patch`.